src/server/config-dist.json

```diff
@@ -9,6 +9,10 @@
 	"SENSORS": {
 	},
 	"SERIAL_PORTS": [],
+	"VRX_SERVER": {
+		"HOST": "localhost",
+		"ENABLED": false
+	},
```

Comment:
I don't think this should be in the 'config-dist.json' file (because this is an optional component that will only be used by a few).

Reply:
Also it's wrong. The key changed to VRX_CONTROL.

Reply:
Removed this section and updated documentation.